atom/common/api/atom_bindings.cc

```diff
@@ -175,6 +177,16 @@ v8::Local<v8::Value> AtomBindings::GetProcessMemoryInfo(v8::Isolate* isolate) {
   return dict.GetHandle();
 }
 
+// static
+v8::Local<v8::Value> AtomBindings::GetCreationTime(v8::Isolate* isolate) {
+  auto timeValue = base::CurrentProcessInfo::CreationTime();
+  if (timeValue.is_null()) {
+    return v8::Null(isolate);
+  }
+  double jsTime = timeValue.ToJsTime();
+  return v8::Number::New(isolate, jsTime);
```

Comment:
Can we not do v8 conversions manually, `native_mate` will handle the conversions in a consistent way if you just return a cpp type. 

I.e. Just make this return `double`